index.bs

```diff
@@ -2775,45 +4881,120 @@ partial interface MLGraphBuilder {
     tensor is specified by the *newShape* argument.
 </div>
 
-### The sigmoid() method ### {#api-mlgraphbuilder-sigmoid}
+<details open>
+  <summary>
+    The {{MLGraphBuilder/reshape(input, newShape)}} steps are:
+  </summary>
+  <div algorithm=reshape class=algorithm-steps>
+    1. If |input| is not an instance of {{MLOperand}}, then throw a "{{TypeError}}" {{DOMException}} and stop.
+    1. Let |outputShape| be an empty array of {{unsigned long}}.
+    1. If |newShape| is a scalar [=number=], set |outputShape| to `[ 1 ]`.
+    1. Otherwise, if |newShape| is an array of {{unsigned long}}:
```

Comment:
According to above explanation of *newShape* argument, here need modify to **1. Otherwise, if |newShape| is an array of {{nullable}} {{unsigned long}}**.

Reply:
My understanding was that we are doing the mapping in this algorithm, so the shape remains an array...
Should this change, let's do it outside of this PR.

Reply:
Made #452